depthai_ros_driver/src/conversion.cpp

```diff
@@ -0,0 +1,92 @@
+#include "depthai_ros_driver/conversion.hpp"
+
+cv::Mat rr::chw2hwc(const cv::Mat& mat) {
+    cv::Mat new_mat = cv::Mat(mat.size(), mat.type());
+    const auto WH = mat.cols * mat.rows;  // WH = (W * H)
+    for (int hWw = 0; hWw < WH; ++hWw) {
+        // hWw = h * (W) + w
+        auto& pt = new_mat.at<cv::Vec3b>(hWw);
+        for (int c = 0; c < 3; ++c) {
+            // essentially hwc = c * (W * H) + h * (W) + w
+            pt[c] = *(mat.data + c * WH + hWw);
+        }
+    }
+    return new_mat;
```

Comment:
Is there some way we can test this?

Can we add tests for at least round trip identity property?

Reply:
Triple trip(chw -> hwc -> wch -> chw)? sure haha